applications/pfem_2_application/custom_utilities/pfem2_utilities.h

```diff
@@ -1243,7 +1243,7 @@ class Pfem2Utils
 	    for (int i=0; i<n_nodes; i++)
         {
 	
-            if(im->GetGeometry()[i].FastGetSolutionStepValue(IS_INTERFACE)>0.5) n_int+=1.0;//im->GetGeometry()[i].FastGetSolutionStepValue(IS_INTERFACE);   
+            if(im->GetGeometry()[i].FastGetSolutionStepValue(IS_INTERFACE)==1) n_int+=1.0;//im->GetGeometry()[i].FastGetSolutionStepValue(IS_INTERFACE);   
```

Comment:
my suggestion would be that you employ flags for this... it would be faster and occupy less memory